src/VideoPlayingScreen/PlayVideo.js

```diff
@@ -83,6 +69,15 @@ export function PlayVideo() {
       </svg>
     );
   }
+function addToPlaylistClickHandler(){
+ const ifIdExist = state.customplaylists[0].videos.find((item)=>item.itemFound.id === itemFound.id)
+  if(ifIdExist){
+  console.log("Video already added to watch later playlist...",state.customplaylists[0])
```

Comment:
functions can be extracted into a different file.
`SVG` can be extracted into different file usually inside the `assets` folder.